<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base_K3bMiscOptionTab</class>
 <widget class="QWidget" name="base_K3bMiscOptionTab">
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="m_checkSaveOnExit">
        <property name="toolTip">
         <string>Ask to save modified projects on exit</string>
        </property>
        <property name="text">
         <string>&amp;Ask to save projects on exit</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Default Temporary Directory:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="m_editTempDir">
          <property name="toolTip">
           <string>The directory where K3b stores temporary files</string>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;This is the default temporary directory. This is where K3b will store temporary files such as iso images or decoded audio files.&lt;p&gt;Be aware that the temporary directory may also be changed in every project burn dialog.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="m_checkSystemConfig">
        <property name="toolTip">
         <string>Check system Configuration</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked K3b will check the system configuration for any problems on startup and when the user changes the settings.</string>
        </property>
        <property name="text">
         <string>&amp;Check system configuration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>GUI Settings</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="m_checkShowProgressOSD">
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked K3b will display the progress in an OSD which always stays on top of all other windows.</string>
        </property>
        <property name="text">
         <string>Show progress &amp;OSD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkHideMainWindowWhileWriting">
        <property name="toolTip">
         <string>Hide the main window while displaying the progress window</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked K3b will hide the main window while displaying the progress dialog.</string>
        </property>
        <property name="text">
         <string>Hide &amp;main window while writing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkShowSplash">
        <property name="toolTip">
         <string>Show the splash screen when K3b starts</string>
        </property>
        <property name="text">
         <string>Show splash screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkKeepDialogsOpen">
        <property name="toolTip">
         <string>Do not close action dialogs after finishing the process</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;If this option is checked K3b will not close action dialogs such as the CD Copy dialog after the process has been finished. It will be kept open to start a new process, for instance, copying another CD.</string>
        </property>
        <property name="text">
         <string>&amp;Keep action dialogs open</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="text">
           <string>&amp;Default action dialog settings:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>m_comboActionDialogSettings</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="K3b::IntMapComboBox" name="m_comboActionDialogSettings" native="true">
          <property name="toolTip">
           <string>Settings to load when opening an action dialog</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>K3b::IntMapComboBox</class>
   <extends>QWidget</extends>
   <header location="global">k3bintmapcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">k3bintmapcombobox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
